lib/middleware/request-handler.ts

```diff
@@ -323,16 +326,24 @@ export function onPoll(req: Interface.IOurRequest,
         return next(new restify.BadRequestError('No active subscriptions.'));
     }
 
+    req.checkQuery('pollid', 'Invalid pollid').isInt();
```

Comment:
If we have a separate middleware to handle checking query parameters, we can configure it to check if `pollid` present and let the route handle validating the content.